src/server.js

```diff
@@ -96,6 +96,7 @@ function initialisExpressSession(app) {
     genid: () => uuid(),
     store: new PgSession({
       conString: config.PUBLIC_SITE_DB_CONNSTR,
+      ttl: 60 * 60,
```

Comment:
Placed TTL of an hour here, to match the cookie maxAge. Will need to double check the effects.